db/migrate/20150405012126_remove_approval_status_from_reservation.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveApprovalStatusFromReservation < ActiveRecord::Migration
+  def change
+    remove_column :reservations, :approval_status, :string
```

Comment:
Good catch; you can see the [migration]() for adding notes to equipment items for an example of a migration that also interacts with your database. Here, however, you'd have to transfer the information *before* you remove the column :stuck_out_tongue:.